src/components/plots/gridPlot.ts

```diff
@@ -12,18 +12,23 @@ export module Plot {
      * grid, and the datum can control what color it is.
      *
      * @constructor
-     * @param {Scale.Ordinal} xScale The x scale to use.
-     * @param {Scale.Ordinal} yScale The y scale to use.
+     * @param {Scale.AbstractScale} xScale The x scale to use.
+     * @param {Scale.AbstractScale} yScale The y scale to use.
      * @param {Scale.Color|Scale.InterpolatedColor} colorScale The color scale
      * to use for each grid cell.
      */
-    constructor(xScale: Scale.Ordinal, yScale: Scale.Ordinal, colorScale: Scale.AbstractScale<any, string>) {
+    constructor(xScale: Scale.AbstractScale<any, any>, yScale: Scale.AbstractScale<any, any>,
+      colorScale: Scale.AbstractScale<any, string>) {
```

Comment:
We could increase the character limit... but nah.  Maybe put an arg on each line...?  Again NBD